docs/decisions/0006-middleware-support-for-http-clients.rst

```diff
@@ -0,0 +1,41 @@
+Middleware Support for HTTP client
+==========================================
+
+Status
+------
+
+Proposed
+
+Context
+-------
+
+We currently expose a HTTP client(axios instance) via ``getAuthenticatedHttpClient`` and ``getHttpClient`` used in
+our MFEs to make HTTP requests. Since Open edX uses Django for its backend services, the MFEs have to handle snake-cased <-> camelCase
+conversions manually. The `axios-case-converter <https://www.npmjs.com/package/axios-case-converter>` package provides a middleware that would handle the conversion automatically. In the
```

Comment:
I believe the parsed link formatting here is needs some TLC (might be a missing `_` or something)